package.json

```
@@ -1,17 +1,26 @@
 {
   "name": "@8base/functions-types",
-  "version": "1.0.1",
+  "version": "1.1.0",
   "description": "Types for 8base Custom Functions",
   "author": "8base",
   "license": "MIT",
+  "type": "module",
   "repository": "https://github.com/8base/functions-types",
-  "types": "types/index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
+    "build": "tsc",
     "typecheck": "tsc --noEmit",
     "postversion": "git push --follow-tags"
   },
   "dependencies": {
     "@graphql-typed-document-node/core": "^3.2.0",
-    "graphql": "^15.8.0"
+    "graphql": "^16.6.0"
+  },
+  "devDependencies": {
+    "typescript": "^5.0.4"
```

Almost everywhere we currently use this version

```suggestion
    "typescript": "^4.3.5"
```